src/test/java/com/exasol/adapter/dynamodb/DynamodbAdapterRefreshTestLocalIT.java

```diff
@@ -0,0 +1,104 @@
+            .withNetwork(NETWORK).withExposedPorts(8888).withLogConsumer(new Slf4jLogConsumer(LOGGER));
+    private static final String TEST_SCHEMA = "TEST";
+    private static final String DYNAMODB_CONNECTION = "DYNAMODB_CONNECTION";
+    private static DynamodbTestUtils dynamodbTestUtils;
+    private static ExasolTestUtils exasolTestUtils;
+
+    /**
+     * Creates a Virtual Schema in the Exasol test container accessing the local DynamoDB.
+     */
+    @BeforeAll
+    static void beforeAll() throws DynamodbTestUtils.NoNetworkFoundException, SQLException, InterruptedException,
+            BucketAccessException, TimeoutException {
+        dynamodbTestUtils = new DynamodbTestUtils(LOCAL_DYNAMO, NETWORK);
+        exasolTestUtils = new ExasolTestUtils(EXASOL_CONTAINER);
+        exasolTestUtils.uploadDynamodbAdapterJar();
+        exasolTestUtils.createAdapterScript();
+        exasolTestUtils.createConnection(DYNAMODB_CONNECTION, dynamodbTestUtils.getDynamoUrl(),
+                dynamodbTestUtils.getDynamoUser(), dynamodbTestUtils.getDynamoPass());
+    }
+
+    @AfterAll
+    static void afterAll() {
+        NETWORK.close();
+    }
+
+    /**
+     * In this test case the schema mapping is replaced but {@code REFRESH} is not called. Thus the virtual schema
+     * should not change.
+     */
+    @Test
+    public void testSchemaDefinitionDoesNotChangeUntilRefresh()
+            throws SQLException, InterruptedException, BucketAccessException, TimeoutException {
+        exasolTestUtils.uploadMapping("basicMapping.json", "mappings/test.json");
+        exasolTestUtils.createDynamodbVirtualSchema(TEST_SCHEMA, DYNAMODB_CONNECTION,
+                "/bfsdefault/default/mappings/test.json");
+        final Map<String, String> columnsBefore = exasolTestUtils.describeTable(TEST_SCHEMA, "BOOKS");
+        exasolTestUtils.uploadMapping("toJsonMapping.json", "mappings/test.json");
+        final Map<String, String> columnsAfter = exasolTestUtils.describeTable(TEST_SCHEMA, "BOOKS");
+        assertThat(columnsBefore, equalTo(columnsAfter));
+    }
+
+    /**
+     * In this test case the schema mapping is replaced and {@code REFRESH} is called. Thus the virtual schema should
+     * have change.
+     */
+    @Test
+    public void testSchemaDefinitionChangesOnRefresh()
+            throws SQLException, InterruptedException, BucketAccessException, TimeoutException {
+        exasolTestUtils.uploadMapping("basicMapping.json", "mappings/test.json");
+        exasolTestUtils.createDynamodbVirtualSchema(TEST_SCHEMA, DYNAMODB_CONNECTION,
+                "/bfsdefault/default/mappings/test.json");
+        final Map<String, String> columnsBefore = exasolTestUtils.describeTable(TEST_SCHEMA, "BOOKS");
+        exasolTestUtils.uploadMapping("toJsonMapping.json", "mappings/test.json");
+        Thread.sleep(5000);// Wait for bucketfs to sync
+        exasolTestUtils.refreshVirtualSchema(TEST_SCHEMA);
```

Comment:
Utils doesn't sound good for the class name. Any other idea for a name?

Reply:
How about:
* ExasolTestInterface
* ExasolTestCopntroler
The class contains mathods that facailitate testing with the Exasol test container. Mostlz methods wrapping SQL statements

Reply:
Maybe:
ExasolTestContainersHandler
ExasolTestContainersManager